spec/controllers/controller_shared_examples.rb

```diff
@@ -0,0 +1,16 @@
+RSpec.shared_examples 'it gracefully handles bad search parameters' do |test_description, bad_parameters, expected_redirect, plural_search_items|
```

Comment:
Did you mean to use this in the tests below?  Or, perhaps it is intended to be used for another PR in which we rescue the error produced by bad params and redirect to some place?

Reply:
it was left over from my first version.  deleted now.